ynput/ayon-ftrack
# Chore: Remove Python 2 vendor

## Changelog Description
Remove Python 2 vendor files.

## Additional info
Python 2 is not supported for a long time so it is not necessary to keep the files there.

## Testing notes
1. Make sure to use the addon (create package & upload, or point dev path to client).
2. Start application.
3. Ftrack should be working e.g. publish plugins should be loaded.
